### ISSUE TITLE
subtitle in the movie

### ISSUE THREAD

**Issue by leiwen83** [sniklaus/softmax-splatting#28]
Hi,

Lots of interpolation method suffers from the subtitle issue, which in video like movie accompany with subtitle, while has great motion between two frame. It would cause subtitle area become blurry or complete damaged.

I wonder know whether softmax splatting could handle well in this case?

Thx,
Lei

**Comment by sniklaus** [comment]
I don't think softmax splatting isn't particularly better than other frame interpolation papers at handling subtitles. However, if one were to supervise our approach with training data subject to subtitles then it should also yield better results in these scenarios than without such training data. This applies to other frame interpolation approaches as well though.
